index.html

```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
             </ul>
         </section>
     </div>
+    <div class="works">
+        <section class="portfolio">
+            <div>
+                <img src="Assets/Tonic1.png" alt="Portfolio image">
+            </div>
+            <div>
+                <h1>Tonic</h1>
+                <div class="summary">
+                    <h2 class="canopy">CANOPY</h2>
+                    <div class="counter"></div>
+                    <h2 class="caption">Back End Dev</h2>
+                    <div class="counter"></div>
+                    <h2 class="caption">2015</h2>                
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="techlist">
+                    <li class="tech">HTML</li>
+                    <li class="tech">CSS</li>
+                    <li class="tech">JavaScript</li>
+                </ul>
+                <button class="project">See project</button>
+            </div>
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), kindly implement the button by following the button styling: `Active`, `Hover` and `Pressed` 🤗💪

(see design) 

![image](https://github.com/firepicaso/RWD_Portfolio/assets/78251927/74d5d6a3-7adf-48ad-b973-06b595c5ae36)